woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -981,13 +958,14 @@ def test_dataframe_methods_on_accessor_returning_series(sample_df):
     assert schema_df.ww.name == 'test_schema'
     pd.testing.assert_series_equal(dtypes, schema_df.dtypes)
 
-    memory = schema_df.ww.memory_usage()
-    assert schema_df.ww.name == 'test_schema'
-    pd.testing.assert_series_equal(to_pandas(memory), to_pandas(schema_df.memory_usage()))
+    if ks and not isinstance(sample_df, ks.DataFrame):
+        # memory_usage() not supported with Koalas
```

Comment:
Is there a method call that returns a series that does work for Koalas? the `dtypes` call covers the non callable case, but if possible, I'd like to cover all three DataFrame types with the callable case as well. (maybe DataFrame.transform?)

Reply:
Updated to use `DataFrame.all()` instead, which works with all three.